Prog/Neighborhood change/4.1_prediction_OTR.R

```diff
@@ -0,0 +1,338 @@
+## predicting displacement risk with OTR based neighborhood change results
+## Yipeng Su
+## last updated 9/25/2024
+
+# Install pacman if not already installed
+if (!require("pacman")) install.packages("pacman")
+
+# Load the required packages using pacman
+pacman::p_load(
+  tidyverse, DescTools, purrr, tidycensus, mapview, stringr, educationdata, sf, 
+  readxl, urbnthemes, sp, ipumsr, survey, srvyr, dplyr, Hmisc, haven, caret, 
```

Comment:
Could not load the urbnthemes package.

Reply:
these are where OTR sales data is not available and I manually coded them when there is data available in previous year, if not the following year data